### PR TITLE
KO translations for validation.md and congratualtion.md.

### PR DESCRIPTION
**Pull request by captain-lee**
Sample screenshot of translated:

### CODE REVIEW

**Comment by captain-lee** [comment]
@techhtml, Please review this PR at your convenience.

File: content/docs/design/visual_story/congratulations@ko.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- 독자의 관심을 끌기 위해 스토리의 마지막에 연관된 다른 콘텐츠로의 링크를 추가해보았습니다.
+
+## 다음 단계
+
+AMP 스토리에 대한 좀 더 자세한 자료는 다음과 같습니다:
+
+- [amp-story 레퍼런스 문서](/ko/docs/reference/components/amp-story.html)
+- [AMP 스토리 베스트 프랙티스](/ko/docs/design/amp_story_best_practices.html)
+
+{% call callout('Note', type='note') %}
```
Comment (bpaduch) [inline content/docs/design/visual_story/congratulations@ko.md]: Can you translate "Note"?  (The 1st string, the second one is a type variable that must remain as is.)

**Comment by bpaduch** [comment]
Syntax LGTM.  Will wait on @techhtml  to review language.

File: content/docs/design/visual_story/congratulations@ko.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+$title: 축하합니다!
+---
+
+AMP 스토리를 완성했습니다!
+
+본 자습서를 통해 배운 것들은 다음과 같습니다:
```
Comment (techhtml) [inline content/docs/design/visual_story/congratulations@ko.md]: 자습서를 튜토리얼로

File: content/docs/design/visual_story/validation@ko.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+$title: AMP HTML 유효성 검사
+---
+
+AMP 페이지를 작성하고 나면 꼭 AMP HTML 유효성 검사를 하십시오. [유효성 검사를 하는 방법에는 여러가지가 있습니다](/ko/docs/fundamentals/validate.html). 이번 자습서에서는 개발자 모드를 켜서 AMP 유효성 검사기(AMP Validator)를 사용해보겠습니다. 다음의 프래그먼트를 URL에 추가하고 새로고침을 하면 개발자 모드가 켜집니다.
```
Comment (techhtml) [inline content/docs/design/visual_story/validation@ko.md]: 자습서 => 튜토리얼

File: content/docs/design/visual_story/congratulations@ko.md (hunk 2 above)
Comment (captain-lee) [reply content/docs/design/visual_story/congratulations@ko.md]: 넵!

File: content/docs/design/visual_story/validation@ko.md (hunk 3 above)
Comment (captain-lee) [reply content/docs/design/visual_story/validation@ko.md]: 옙!

**Pull request merged**